helm/prometheus-rules/templates/alerting-rules/credentiald.rules.yml

```
@@ -1,9 +1,11 @@
+{{- if eq .Values.managementCluster.provider.flavor "vintage" }}
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
   creationTimestamp: null
   labels:
     {{- include "labels.common" . | nindent 4 }}
+    # Kept because it will be gone with Vintage
```

Oops, I had missed one!
```suggestion
    # No need for .Values.mimir.enabled condition - will be gone with Vintage
```